welcome.css

```diff
@@ -72,6 +71,82 @@ html, body {
 
 .text-tag {
   text-align: center;
-  font-size: 32px;
+  font-size: 38px;
   color: olive;
+}
+
+.burger-icon:before {
+  display: none;
+  font-size: 32px;
+}
+
+.click-icon {
+  display: none;
+}
+
+@media (max-width: 800px) {
+  .burger-icon:before{
+    display: block;
+    margin-top: 3%;
+  }
+
+  .menu-btn {
+    position: absolute;
+    right: 0;
+    margin: 8px;
+    z-index: 3;
+  }
+
+  .click-icon:checked ~ .menu-btn i:before{
+    content: "\f00d";
+  }
+
+  .click-icon:checked ~ .menu-btn {
```

Comment:
```
.click-icon:checked ~ .menu-btn i:before{
    content: "\f00d";
  }

  .click-icon:checked ~ .menu-btn {
```
I have never written this code before, so please tell me the purpose of writing these codes?
Is there another way to write it that is easier to understand?

Reply:
".click-icon:checked ~ .menu-btn i:before{
    content: "\f00d";
} "  when someone clicked on the icon after that it should show the close icon. without using, it display like this:

<img width="927" alt="Screenshot 2024-07-10 at 1 27 36 PM" src="https://github.com/scheme-dojo/scheme-dojo.github.io/assets/143992531/150475cd-4b43-4285-8470-8ce6c0e14ebd">

".click-icon:checked ~ .menu-btn {" this is for the style of it, color

Reply:
I see. The state is managed using CSS.
 I learned a lot. thank you.
Well, let's do it with CSS this time. 
Rabia-san is using it after thinking carefully, then the problem here is OK.